beacon-chain/rpc/eth/validator/handlers_block_test.go

```diff
@@ -1000,17 +1009,20 @@ func TestProduceBlockV3(t *testing.T) {
 		jsonBytes, err := json.Marshal(block.Message)
 		require.NoError(t, err)
 		v1alpha1Server := mock2.NewMockBeaconNodeValidatorServer(ctrl)
-		v1alpha1Server.EXPECT().GetBeaconBlock(gomock.Any(), gomock.Any()).Return(
+		v1alpha1Server.EXPECT().GetBeaconBlock(gomock.Any(), &eth.BlockRequest{
+			Slot:         1,
+			RandaoReveal: bRandao,
+			Graffiti:     bGraffiti,
+			SkipMevBoost: false,
```

Comment:
This is the most important line in tests - it asserts the value of `SkipMevBoost`